doc/api/packages.md

```diff
@@ -69,14 +69,14 @@ expressions:
 * Strings passed in as an argument to `--eval`, or piped to `node` via `STDIN`,
   with the flag `--input-type=module`.
 
-* When using [`--experimental-detect-module`][], code containing syntax only
-  successfully parsed as [ES modules][], such as `import` or `export`
-  statements or `import.meta`, having no explicit marker of how it should be
-  interpreted. Explicit markers are `.mjs` or `.cjs` extensions, `package.json`
-  `"type"` fields with either `"module"` or `"commonjs"` values, or
-  `--input-type` or `--experimental-default-type` flags. Dynamic `import()`
-  expressions are supported in either CommonJS or ES modules and would not
-  cause a file to be treated as an ES module.
+* Code containing syntax only successfully parsed as [ES modules][], such as
+  `import` or `export` statements or `import.meta`, having no explicit marker of
+  how it should be interpreted. Explicit markers are `.mjs` or `.cjs`
```

Comment:
IMO "having no explicit marker..." feels out-of-place. I had to read the sentence several times to understand the correlation between the concepts, could this be reworded for clarity?

(I'm aware it was like this before)

Reply:
Maybe this?

```suggestion
* Code containing syntax only successfully parsed as [ES modules][], such as
  `import` or `export` statements or `import.meta`, with no explicit marker of
  how it should be interpreted. Explicit markers are `.mjs` or `.cjs`
```